cyclonedx-bom/src/external_models/normalized_string.rs

```diff
@@ -45,9 +46,15 @@ impl NormalizedString {
     }
 }
 
-impl ToString for NormalizedString {
-    fn to_string(&self) -> String {
-        self.0.clone()
```

Comment:
Removing `ToString` would be a semver break. Are you sure this is justified?

Reply:
Nevermind, an implementation of it is [provided](https://doc.rust-lang.org/stable/std/string/trait.ToString.html) by implementing `Display`.